targets/XPlatCppSdk/source/cppsdk/source/playfab/PlayFabHttp.cpp

```diff
@@ -191,6 +191,7 @@ namespace PlayFab
             Json::CharReader* jsonReader(jsonReaderFactory.newCharReader());
```

Comment:
It seems to me that maybe jsonReader shouldn't be a pointer?
Maybe we should just create jsonReader here as a stack variable instead.
I think the change would be equally as small, and we won't have to figure out how to optimize or wrap a memory allocation later.

Reply:
We discussed this offline, we receive the pointer from library's API newCharReader() and can't control if it's to be a pointer or an object. Resolving this for now.